Better delineation of host-based requirement sets versus Common APIs (#716)
OfficeDev/office-js-docs-reference: docs/docs-ref-autogen/overview/overview.md

The JavaScript API for Office enables you to create web applications that interact with the object models in Office host applications. Use this section to learn more about the classes, methods, and other types available for building Office Add-ins.

> The following is a list of [requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets) for the [supported Office host applications](/office/dev/add-ins/overview/office-add-in-availability). Each item links to a version of the API reference documentation that is supported by that requirement set (e.g. ExcelApi 1.3 shows APIs in ExcelApi 1.1, 1.2, 1.3 as well as the Common APIs). Common API content includes all the [Common API requirement sets](/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets).

Reviewer: ```suggestion
The following is a list of [requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets) for the [supported Office host applications](/office/dev/add-ins/overview/office-add-in-availability). Each item links to a version of the API reference documentation that is supported by that requirement set (for example, ExcelApi 1.3 shows APIs in ExcelApi 1.1, 1.2, 1.3 as well as the Common APIs). Common API content includes all the [Common API requirement sets](/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets).
```

            </div>
        </a>
    </li>
    <li>

>         <a class="card x-hidden-focus" href="/javascript/api/project?view=common-js">

Reviewer: ```suggestion
        <a class="card x-hidden-focus" href="/javascript/api/common?view=common-js">
```

            <div class="cardImageOuter">
                <div class="cardImage">

>                     <img src="/javascript/api/overview/images/logo-project.svg" alt="Project add-ins" />

Reviewer: Either rename the image or use the current image name...
```suggestion
                    <img src="/javascript/api/overview/images/logo-project-server.svg" alt="Project add-ins" />
```

                </div>
            </div>
            <div class="cardText">
                <h3>Project requirement sets</h3>
                <ul>

>                     <li><a style="font-size: 1rem;" href="/javascript/api/powerpoint?view=common-js">Common APIs only</li></p>

Reviewer: ```suggestion
                    <li><a style="font-size: 1rem;" href="/javascript/api/common?view=common-js">Common APIs only</li></p>
```
